### ISSUE TITLE
Hide Category property

### ISSUE THREAD

**Issue by bogdanstojanovic** [Geta/EpiCategories#16]
Are there any way to hide Category property that links to default Episerver categories? 
Scaffolding property does not help. Furthermore, you can't override the property.

**Comment by MattisOlsson** [comment]
That should happen out of the box when installing the package. It doesn't for you?

**Comment by bogdanstojanovic** [comment]
No, as far as you can see on the screenshot below. I can't hide the Category property from Geta based category. 
![image](https://user-images.githubusercontent.com/5629507/56721868-1a697000-6746-11e9-9057-429937cd040d.png)

**Comment by MattisOlsson** [comment]
Do you have any other editor descriptors for CategoryList type in your project?
